modelling/models.py

```diff
@@ -0,0 +1,25 @@
+from tf_idf import get_articles_keywords
+
+class Model:
```

Comment:
This shoud be an abstract class (you can use the abc module in python for that). That way, if we want to implement a new model (training word2vec on our corpus for example), we can inherit the model class and be sure that it will behave nicely in the rest of the pipeline.